cyberdrop_dl/ui/prompts/user_prompts.py

```diff
@@ -0,0 +1,218 @@
+
+    if dry_run:
+        manager.config_manager.settings_data["Browser_Cookies"]["browsers"] = browsers
+        manager.config_manager.settings_data["Browser_Cookies"]["sites"] = domains
+        return
+
+    get_cookies_from_browsers(manager, browsers=browsers, domains=domains)
+    console.print("Import finished", style="green")
+    basic_prompts.enter_to_continue()
+
+
+def browser_prompt() -> str:
+    choices = [Choice(browser, browser.capitalize()) for browser in BROWSERS]
+    return basic_prompts.ask_checkbox(choices, message="Select the browser(s) for extraction:")
+
+
+""" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ V4 IMPORT PROMPTS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"""
+
+
+def import_cyberdrop_v4_items_prompt(manager: Manager) -> int:
+    """Import Cyberdrop_V4 Items."""
+    prompt_header(manager)
+    OPTIONS = [["Import config", "Import download_history.sql"]]
+    choices = basic_prompts.create_choices(OPTIONS)
+    console.print("V4 Import Menu")
+    return basic_prompts.ask_choice(choices)
+
+
+def import_v4_config_prompt(manager: Manager) -> tuple[str, Path] | None:
+    """Asks the user for the name and path of the config to import. Returns `None` if the config name is invalid."""
+    new_config = create_new_config(manager, title="What should this config be called:")
+    if not new_config:
+        return None
+    return new_config, basic_prompts.ask_file_path("Select the config file to import:")
+
+
+def import_v4_download_history_prompt() -> Path:
+    return basic_prompts.ask_file_path("Select the download_history.sql file to import:")
+
+
+""" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ OTHERS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"""
+
+
+def prompt_header(manager: Manager, title: str | None = None) -> None:
+    clear_term()
+    title = title or f"[bold]Cyberdrop Downloader ([blue]V{__version__!s}[/blue])[/bold]"
+    console.print(title)
+    console.print(f"[bold]Current config:[/bold] [blue]{manager.config_manager.loaded_config}[/blue]")
+
+
+""" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ DEPRECATED PROMPTS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"""
+
+
+def donations_prompt() -> None:
+    """Donations prompt."""
+    clear_term()
+    msg = """
+    [bold]Donations[/bold]
+
+    I started making this program around three years ago at this point,
+    It has grown larger than I could've imagined and I'm very proud of it.
+    I have put a lot of time and effort into this program and I'm glad that people are using it.
+    Thanks to everyone that have supported me,
+    it keeps me motivated to continue working on this program
+
+    If you'd like to support me and my work, you can donate to me via the following methods:
+    BuyMeACoffee: https://www.buymeacoffee.com/jbsparrow
+    Github Sponsor: https://github.com/sponsors/jbsparrow
+
+    Thank you for your support!
+    """
+    console.print(dedent(msg))
+    basic_prompts.enter_to_continue()
```

Comment:
```suggestion
```

The donation prompt has my donation info but Jules' donation message, plus we don't even include the donation prompt. Probably best to just remove it.